### PR TITLE
Updating error copy

### PR DESCRIPTION
**Pull request by mattchenderson**
Per design team revision.

Addresses https://github.com/Azure/azure-webjobs-sdk-templates/issues/152

### CODE REVIEW

**Comment by davidebbo** [comment]
:shipit:

File: Bindings/bindings.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
                     "validators": [
                         {
                             "expression": "^[A-Za-z0-9]$|^[A-Za-z0-9][\\w-\\.\\/]*[A-Za-z0-9]$",
-                            "errorText": "Invalid event hub name"
+                            "errorText": "Your event hub name must start and end with a letter or number, and it can contain only lowercase letters, numbers, the hyphen, and the underscore."
                         }
```
Comment (soninaren) [inline Bindings/bindings.json]: Periods are allowed in eventhub names

File: Bindings/bindings.json [hunk 2]
```diff
@@ -71,7 +71,7 @@
...
```
Comment (soninaren) [inline Bindings/bindings.json]: Periods are allowed in eventhub names. And has a 50 character limit.

File: Bindings/bindings.json [hunk 3]
```diff
@@ -226,7 +226,7 @@
                     "validators": [
                         {
                             "expression": "(^[a-z0-9](?:[a-z0-9]|(?:\\-(?!\\-))){1,61}[a-z0-9]$)|(^[a-z0-9](?:[a-z0-9]|(?:\\-(?!\\-))){1,61}[a-z0-9][\\/](\\S){1,1023}$)",
-                            "errorText": "Invalid blob path"
+                            "errorText": "Your blob path must be a container name of 3 to 63 characters, followed by a blob name of 1 to 1,024 characters. The blob name can be any combination of characters and can contain a maximum of 254 path segments, which are separated by the forward slash (/)."
                         }
```
Comment (soninaren) [inline Bindings/bindings.json]: For Trigger it can also be just a container name.

**Comment by soninaren** [comment]
This PR is against master. Please resend another PR against dev. Unless it is on purpose. Text for topic binding also needs to be updated

File: Bindings/bindings.json [hunk 4]
```diff
@@ -102,7 +102,7 @@
                     "validators": [
                         {
                             "expression": "^[0-9a-z][a-z0-9-]{2,62}$",
-                            "errorText": "Invalid queue name"
+                            "errorText": "Your queue name must start and end with a letter or number, and it can contain only lowercase letters, numbers, and the hyphen. The name must be 3 to 63 characters."
                         }
```
Comment (soninaren) [inline Bindings/bindings.json]: Same here period is allowed for Service bus Queue Name and topic name

**Comment by mattchenderson** [comment]
Replaced by https://github.com/Azure/azure-webjobs-sdk-templates/pull/191 which targets dev

**Pull request closed without merging**